site/source/docs/compiling/Deploying-Pages.rst

```diff
@@ -56,7 +56,7 @@ While the first run experience of visiting a page can take some time to finish a
 
 - All browsers have an implementation defined limit (20MB or 50MB) for assets, and files larger than that will bypass the browser's built-in web caches altogether. Therefore it is recommended that large ``.data`` files are manually cached to IndexedDB by the main page. The Emscripten linker option ``--use-preload-cache`` can be used to have Emscripten implement this, although it can be desirable to manually manage this on the html page in a custom manner, since that allows taking control of which database the assets are cached to, and what kind of scheme will be used to evict data from it.
 
-- Compilation results of asm.js modules are cached automatically by the browser, and there is little control over this. WebAssembly on the other hand supports explicit caching of compiled ``WebAssembly.Module`` objects to IndexedDB. This feature should be always leveraged, since it allows skipping the whole compilation process on the second page visit.
+- Compilation results of asm.js modules are cached automatically by the browser, and there is little control over this. WebAssembly does not yet have automatic caching support among all modern browsers, though Chromium-based browsers do (read more `here <https://v8.dev/blog/wasm-code-caching>`_). Previously, manual caching of compiled modules via IndexedDB was recommended; however, this is now largely unsupported (read more `here <https://github.com/WebAssembly/spec/issues/821>`_).
```

Comment:
Can you also remove the first sentence here.. we don't support asm.js anymore.

Reply:
Done.